docs/OpenID4VPCrossDeviceFlow.md

```
@@ -1,56 +1,126 @@
+- Inji Verify generates an authorization request through the backend, which is rendered as a QR code.
+- The QR code is scanned by the wallet to initiate the VP flow.
+  - Required fields for creating an authorization request:
       - client_id
-      - **_presentationDefinitionId_** or **_presentationDefinition_** (presentationDefinition - should adhere to the [Specification](https://identity.foundation/presentation-exchange/spec/v2.0.0)
-      - transactionId (Optional) - will be generated at server if not provided
-  - Inji verify Ui will make a request to backend to create the authorization request and the UI will render this as a QR code for the wallet to scan.
+      - **_presentationDefinitionId_** or **_presentationDefinition_** (presentationDefinition - should adhere to the [Specification 🔗](https://identity.foundation/presentation-exchange/spec/v2.0.0)
+      - transactionId (Optional) - If not provided, it will be generated by the server, base64-encoded, and set as an HTTP-only secure cookie in the response header.
+  - The Inji Verify UI sends a request to the backend to create the authorization request and renders the response as a QR code.
 
 ###  Authorization Request Status:
```

markdownlint-cli2 (0.22.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)